foundations/config.yaml

```diff
@@ -11,8 +11,14 @@ data:
     #    longhorn: true
     #    openebs: true
     #    ingress: true
+    #    klum: true
     #  cloud:
     #     region: ber1
     #     publicIPs:
     #       - cidr: 192.168.181.1/32
     #       - cidr: 192.168.181.2/32
+    #  klum:
```

Comment:
why is all of that user facing?

Reply:
The bootstrap operator has to mainly configure the `serverAddress` and optionally the `contextName`, which are end-user facing (bad serverAddress/default one and users won't be able to log into the cluster with their generated kubeconfig)

Reply:
I'm removing those that should essentially stay as default values though

Reply:
if we cant fetch it from the cluster itself, those needs to be set as a single var, not for every component individually. 

namespace shouldn't be settable by the user, that makes no sense

Reply:
address should just be generated out of intranet domain, which is unfortunately set during cluster creation anyway, because of the k8s cert.

Reply:
contextName is actually our cluster code which also repeats at least in s32s3 backup location, and cortex token

Reply:
I'm going to create proper variables for these instead of putting them under klum then, this is better